lighthouse-core/test/lib/drivers/driver-test.js

```diff
@@ -32,6 +32,8 @@ driverStub.sendCommand = function(command, params) {
       return Promise.resolve({
         nodeId: params.selector === 'invalid' ? 0 : 231
       });
+    case 'Storage.clearDataForOrigin':
+      return Promise.resolve({origin: 'http://aliexpress.com', storageTypes: 'all'});
```

Comment:
the actual command doesn't appear to have a return object, so I don't think this should resolve on anything, just `return Promise.resolve();`